wiki/Game_Modifiers/en.md

```diff
@@ -270,15 +270,15 @@ The method used to increase the speed does not increase the pitch of the song, b
 
 ---
 
-In [osu!taiko][osu!taiko wikilink] mode, since the hit leniency is greatly increased on top of osu!taiko's already strict overall difficulty in addition to making slider ticks a lot harder to hit and reducing the value of dendens, Double Time is usually regarded as the hardest mod in osu!taiko and is seldom used.
+In [osu!taiko][osu!taiko wikilink] mode, since the hit leniency is greatly reduced on top of osu!taiko's already strict overall difficulty in addition to making slider ticks a lot harder to hit and reducing the value of dendens, Double Time is usually regarded as the hardest mod in osu!taiko and is seldom used.
```

Comment:
* remove "a lot", if emphasis is needed recommend "much" or even "considerably" in favor of the less professional-sounding "a lot"
* maybe put "in addition to ... dendens" in parentheses? this sentence is a massive run-on atm